spylon_kernel/init_spark_magic.py

```diff
@@ -37,8 +39,9 @@ def cell_init_spark(self):
 
         globals_dict = self.env
         exec(self.code, globals_dict)
+        application_name = globals_dict.get('application_name', None)
```

Comment:
I'm looking at a diff quickly, but I'm guessing you could just put "ScalaMetaKernel" here in place of the None and remove the check later. Also, it should probably be "spylon-kernel" by default.